float32 motor_FL
float32 motor_FR
float32 motor_BL
float32 motor_BR
float64 mean_speed_FL
float64 mean_speed_FR
float64 mean_speed_BL
float64 mean_speed_BR
int16 encoder_FL
int16 encoder_FR
int16 encoder_BL
int16 encoder_BR